pkg/config/validation/testdata/invalid/bundle.json

```diff
@@ -0,0 +1,32 @@
+{
+    "schema": "olm.bundle",
+    "name": "etcdoperator-community.v0.6.1",
+    "image": "quay.io/operatorhubio/etcd:v0.6.1",
+    "properties":[
+        {
+            "type": "olm.package.provided",
+            "value": {
+                "version": "0.6.1"
+            }
+        },
+        {
+            "type":"olm.gvk.provided",
+            "value": {
+                "group": "etcd.database.coreos.com",
+                "kind": "EtcdCluster",
+                "version": "v1beta2"
+            }
+        },
+        {
+            "type": "olm.channel",
+            "value": {
+                "name": "alpha"
+            }
+        }
+    ],
+    "relatedImages": [
+        {
+            "image": "quay.io/coreos/etcd-operator@sha256:bd944a211eaf8f31da5e6d69e8541e7cada8f16a9f7a5a570b22478997819943"
+        }
+    ]
+}
```

Comment:
consider using something like https://golang.org/pkg/testing/quick/ to write property based/generative tests as well